internal/component/database_observability/mysql/collector/schema_table_test.go

```diff
@@ -34,7 +34,7 @@ func TestSchemaTable(t *testing.T) {
 		collector, err := NewSchemaTable(SchemaTableArguments{
 			DB:              db,
 			InstanceKey:     "mysql-db",
-			CollectInterval: time.Second,
+			CollectInterval: time.Millisecond,
```

Comment:
just curious, how come the collect interval had to be changed?

Reply:
This is just a drive-by improvement. Tests that expect to loop twice were sitting idle for 1 second. I changed everything down to 1ms so that tests that loop twice will complete much faster (this shaved ~1s from tests duration as the test cases run in parallel anyway).